tailwind.config.js

```diff
@@ -3,6 +3,11 @@ module.exports = {
   content: ["./*.{html,js}", "./src/**/*.{html,js}"],
   theme: {
     extend: {
+      fontSize: {
+        navsize: "0.9375rem",
```

Comment:
Using camel case in as naming will easy code base readability as well as stick to the consistency.